src/main/java/org/folio/rest/impl/PostGobiOrdersHelper.java

```diff
@@ -340,7 +340,22 @@ public PostGobiOrdersHelper(HttpClientInterface httpClient,
 
   public CompletableFuture<String> lookupMock(String data) {
     logger.info("Mocking the data lookup for: " + data);
-    return CompletableFuture.completedFuture(UUID.randomUUID().toString());
+    return CompletableFuture.completedFuture(UUID.randomUUID()
+      .toString());
+  }
+
+  public CompletableFuture<String> separateISBNQualifier(String data) {
+    if (data.split("\\s+").length > 1) {
+      return completedFuture(data.split("\\s+")[1]);
+    }
+    return completedFuture(null);
+  }
+
+  public CompletableFuture<String>  truncateISBNQualifier(String data) {
+    if(data.split("\\s+").length > 1) {
+      return completedFuture(data.split("\\s+")[0]);
```

Comment:
If the first character is a space, then after splitting the string, the `productId` field will get an empty string and the `qualifier` will get ISBN number.

Reply:
done